hangups/ui/__main__.py

```diff
@@ -198,11 +198,23 @@ def _on_event(self, conv_event):
 
     def _on_quit(self):
         """Handle the user quitting the application."""
-        future = asyncio.async(self._client.disconnect())
-        future.add_done_callback(lambda future: future.result())
+        asyncio.ensure_future(self._client.disconnect())
```

Comment:
reverted in https://github.com/tdryer/hangups/pull/343/commits/48b96a82af7f9ee93a55548060fa2c8114c9fb69
I will keep the[ ``drop-asyncio-async`` branch](https://github.com/das7pad/hangups/tree/drop-asyncio-async) updated and ready for a future merge.